### ISSUE TITLE
Хамелеон Кит не работает на кепки

### ISSUE THREAD

**Issue by AgRevol** [TauCetiStation/TauCetiClassic#13462]
#### Что должно было произойти
Вы надеваете хамелеон кит. Вы выбираете в списке, например, Rainbow Cap. У вас на голове теперь радужная кепка.
#### Что произошло на самом деле
Кепки нет. 
![image](https://github.com/user-attachments/assets/21abbe00-06f7-4d54-98fb-45e2b82fe82c)
![image](https://github.com/user-attachments/assets/d279c5b6-2670-454a-a835-517e8bb752be)

### Шаги воспроизведения

1. Получить хамелеон кит.
2. Поменять внешность шляпе на любую кепку.
3. Надеть.

### Дополнительная информация:

Думаю, это связано с ворлд/инхенд стейтами. На голове отображается словно ворлд стейт.
![image](https://github.com/user-attachments/assets/d6cba12e-6e61-4ba9-b1c8-cdfbae47ea56)

**Comment by Udokun** [comment]
Да, почему-то все спрайты с инворлд спрайтами подтягиваются к этим предметам.
